liquidity/liquidity.go

```diff
@@ -508,6 +590,18 @@ func (m *Manager) makeLoopOutRequest(suggestion *LoopOutRecommendation,
 		MaxPrepayAmount:     quote.PrepayAmount,
 		SweepConfTarget:     m.params.ConfTarget,
 	}
+
+	if autoOut {
+		request.Label = labels.AutoOutLabel()
+
+		addr, err := m.cfg.Lnd.WalletKit.NextAddr(ctx)
+		if err != nil {
+			return loop.OutRequest{}, err
+		}
+		request.DestAddr = addr
```

Comment:
Isn't it that no address defaults to generating an address when the swap is launched? Or otherwise the returned requests from `SuggestSwaps` won't work (no dest addr)

Reply:
`SuggestSwaps` would be taken by the user and passed to the rpc server (which fills the addr), we pass directly to the swap client to bypass the label check (which exists on rpc server level) so we need to fill the addr for autoloop (but not for suggestions).